ui/pages/setup_secure.tsx

```diff
@@ -21,7 +21,7 @@ const SetupSecure: NextPage = () => {
   // Get ?flow=... from the URL
   const router = useRouter();
   const {
-    return_to: returnTo,
+    return_to: _returnTo,
```

Comment:
```suggestion
    return_to,
```
I think we can keep it simpler and avoid the aliasing here?